packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -165,27 +203,18 @@ def _get_tx_data(
                 "send_raw_transaction unsuccessful! Insufficient funds."
             )
 
+        # Trigger repricing mechanism if we have succesfully detected the strategy price
+        # and we receive an underpriced status.
```

Comment:
```suggestion
        # Store gas price if we have successfully detected the strategy and we receive an underpriced status, 
        # so that the repricing mechanism is triggered next time.
```